dataMining/src/database/fixtures/load.ts

```diff
@@ -0,0 +1,42 @@
+import * as path from 'path';
+import { 
```

Comment:
to keep things DRY in future, we will need some common code, this is in server component as well, I believe